Roosevelt Rondon Scientific Expedition

joelAVS opened the pull request:
A minor event chain describing the Roosevelt Rondon Scientific Expedition, which took place from 1913 to 1914, if extra flavor is activated, there's 1 in 10 chance of Roosevelt actually dying. This includes new historical pictures for the events and new newspaper entries.

joelAVS committed:
Basic Event Added

joelAVS committed:
RoseveltRondon pt2 added

joelAVS committed:
Good ending, fleshing out

The 2 first parts of the event are finished and polished, added good ending and basis for bad ending

joelAVS committed:
random chance adjusted

Random chance adjusted, ted dies event added and polished

joelAVS committed:
final commit

Finished localization and pictures, ajusted time frame, added fantasy check for roosevelt death, added proper newspaper entries,

joelAVS committed:
Bug fix newspaper

typo on newspaper

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,155 @@
+				tag = KPB
+			}
+			year = 1913
+			has_country_flag = rosevelt_rondon
```
Put this trigger first, less likely to be true than a tag = check, not to mention the tag = is doing a double check cuz of the OR

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,155 @@
+			}
+			year = 1914
+			month = 4
+			has_country_flag = rosevelt_rondon_ending
```
See previous comment

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,155 @@
+			}
+			year = 1914
+			month = 4
+			has_country_flag = rosevelt_rondon_ending
```
See first comment

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,155 @@
+			prestige = -3
+			relation = { who = USA value = -15 }
+			#USA = { country_event = Ted_Dies_Event }
+			clr_country_flag = rosevelt_rondon_ending
```
This has to be an immediate to prevent both endings firing.

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,155 @@
+			prestige = 2
+			relation = { who = USA value = +10 }
+			#USA = { country_event = Ted_Returns_Event } 
+		    clr_country_flag = rosevelt_rondon_ending
```
This has to be an immediate to prevent both endings firing rather than one.

GeneralVonGelre replied:
This shouldn't be double removed, just in immediate is enough.

joelAVS committed:
Event check optmization

Removing unnecessary event checks, organizing events check order and adding an immediate = {} function to stop both endings from firing at once

joelAVS committed:
Event optmization 2

For some reason I assumed the US wasnt a democracy most of the times.
 Philosophically I may be right, but not mechanically speaking.

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,145 @@
+		id = 57000
+
+		trigger = {
+			year = 1913
```
Year = shouldnt be first as its checked and either true or false for all tags

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,145 @@
+		fire_only_once = yes
+
+		trigger = {
+			year = 1914
```
Check country flag before all this stuff

GeneralVonGelre commented on TGC/events/BRZFlavor.txt:
```diff
@@ -4294,3 +4294,145 @@
+		fire_only_once = yes
+
+		trigger = {
+			NOT = { has_global_flag = extra_flavour_disabled }  #IRL he survived, so this bad ending should count as fantasy
```
Check country flag first

joelAVS committed:
event check optmization 2

joelAVS committed:
Double flag clear removed
